allure-generator/src/main/javascript/helpers/timeago.js

```diff
@@ -0,0 +1,8 @@
+import moment from 'moment';
```

Comment:
Also I would suggest using `date-fns` instead of `moment`.
https://date-fns.org/docs/distanceInWordsToNow

The API is much simpler

```js
import distanceInWordsToNow from 'date-fns/distance_in_words_to_now';
import format from 'date-fns/format';

const threshold = 24 * 3600 * 1000;

export default function (date) {
     const dateString = Date.now() - date < threshold ? distanceInWordsToNow(date) : format(date, 'DD MMMM YYYY');
     return new runtime.SafeString(`<span title="${format('DD MMMM YYYY, H:mm:ss')}">${dateString}</span>`);
 } 
```